src/components/HomePage/HomepageTestimonials.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="testimonials">
+    <div layout="column u4">
+      <div v-for="t in testimonials" :key="t.name" layout="row u1">
+        <span flex
+          ><i>“{{ t.text }}”</i></span
+        >
+        <span flex
+          ><b>{{ t.name }}</b> ({{ t.institute }})</span
```

Comment:
```suggestion
        <span><b>{{ t.name }}</b> ({{ t.institute }})</span 
```

All sibling elements with `flex` will split any remaining (not taken by non-flex elements) space evenly between each other.

Either the `flex` on the name is not necessary, as you don't want the name to always take half of the space, or you can get rid of both `flex`es and keep the element's natural height for both text and name.